src/ocean/net/device/Berkeley.d

```diff
@@ -1353,7 +1355,14 @@ deprecated public class IPv4Address : Address
         static uint parse(istring addr)
         {
                 char[64] tmp;
-                return ntohl(inet_addr(toStringz(addr, tmp)));
+                if ((addr.length + 1) > tmp.length)
```

Comment:
Why not simply replace stack-allocated `char[64] tmp` with `static char[] tmp` and use `copy` + `ocean.text.utill.StringC` instead? That was what I have anticipated as a migration instruction, though in this specific case I liked preserving exact old semantics because it is untested scary legacy code.